src/database/seeding/seeding.service.ts

```diff
@@ -41,6 +43,40 @@ export class SeedingService {
           throw new Error('Failed to create all users');
         }
 
+        const or1 = organisationRepository.create({
+          name: 'Org 1',
+          description: 'Description 1',
+          email: 'test1@email.com',
+          industry: 'industry1',
+          type: 'type1',
+          country: 'country1',
+          state: 'state1',
+          address: 'address1',
+          owner: savedUsers[0],
+          creator: savedUsers[0],
+          isDeleted: false,
+        });
+
+        const or2 = organisationRepository.create({
+          name: 'Org 2',
+          description: 'Description 2',
+          email: 'test2@email.com',
+          industry: 'industry2',
+          type: 'type2',
+          country: 'country2',
+          state: 'state2',
+          address: 'address2',
+          owner: savedUsers[0],
+          creator: savedUsers[0],
```

Comment:
What's the point of seeding this data? Is this initial seed when server freshly loads?

Reply:
yes, it's initial seeding